### DISCUSSION TITLE
Highlight closed doors in pyramid plunder

### DISCUSSION THREAD

**Discussion by jarricvankrieken** [runelite/runelite#15419] [Q&A] [answered]
It would be really nice to have highlighted doors that are still closed in pyramid plunder. Now you will have to turn your camera to see if they are closed or not. Sometimes it can be really frustrating when you've been at a door and failed to open it, but did not see it fail. I know there is a chat that says opening the door failed, but a visual indicator would be so much better.

**Comment by Enriath** [comment, marked as answer]
The core Pyramid Plunder plugin already does this.
![image](https://user-images.githubusercontent.com/2979691/181853077-3f45e707-1485-459d-abe1-3cacc4b5157a.png)
